source/fundamentals/monitoring.txt

```diff
@@ -38,7 +38,7 @@ deployment:
 Event Descriptions
 ------------------
 
-The following table lists all the SDAM events that are made available to your
+The following table lists all the SDAM events that are available to your
```

Comment:
This sentence sounds wrong to me. I would suggest:
```suggestion
The following table lists all the SDAM events made available to your
```

Reply:
Updated to be more direct:

> You can subscribe to any of the following SDAM events: